### DISCUSSION TITLE
GSoC: Reaching out to mentors of NNCF Quantization Analytics Tool !

### DISCUSSION THREAD

**Discussion by noelp2500** [openvinotoolkit/openvino#23152] [Google Summer of Code]
Hi OpenVino Developers,

I'm Noel, a first year graduate student (masters) at University of California in EECS and I would love to know more about the project [NNCF Quantization Analytics Tool](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#9-nncf-quantization-analytics-tool)

I was informed about OpenVino and its tools at Samsung Next Hackathon in San Francisco by @raymondlo84.

As a graduate student actively working at [PADSYS](https://padsys.org/publications.html) Lab on Federated learning especially pertaining to faster model inference, and with my prior background building analytics tools (SDE at HaiX where we built a social media analytics tool) and my prior [research](https://noel-vs-code-resume-public-version-pereiranoel1225-gmailcom.vercel.app/#/researchpapers), I believe this project falls under my domain of interest, and I would love to be mentored and contribute!

As for the prerequisite task, I'm already in the process of completing one that I picked, but will do any other if necessary!

I would like to get in touch with the project mentors (Alexander Suslov, Andrey Churkin - @alexsu52 @andrey-churkin) and would appreciate their advice on how to best prepare my application for submission!

Best,
Noel Pereira

**Comment by alexsu52** [comment]
Hi @noelp2500, thank you for your interest in the project!

It's good that you have already taken on some task. The GFI is a test task and it makes no difference in which OpenVINO toolkit project you are doing it. Please share the issue link here.

If you have any questions, please feel free to let me know.
